CHANGELOG.rst

```diff
@@ -2,8 +2,16 @@
 Changelog
 =========
 
-TBR
----
+0.7.0 (to be released)
+----------------------
+
+* Support :ref:`page object dependency serialization <dep-serialization>`, to
+  make it easier to implement caching mechanisms in web-poet frameworks.
```

Comment:
Is it the main use case for serialization?

Reply:
This line was added before the testing framework was merged. Do you think we need to keep it at all as it's a part of the testing feature?

Reply:
What about https://github.com/scrapinghub/web-poet/pull/111/commits/f0657ea820780786d5e73503fefc640966bbcda8 ?